Document LCP and INP subparts breakdown for Browser RUM (#36286)
DataDog/documentation: content/en/real_user_monitoring/application_monitoring/browser/monitoring_page_performance.md

- For First Input Delay, RUM reports the CSS selector of the first element the user interacted with.
- For Cumulative Layout Shift, RUM reports the CSS selector of the most shifted element contributing to the CLS.

### Diagnose Core Web Vitals with subparts

> 

Reviewer: ```suggestion
**Note**: These attributes require Browser SDK v6.32.0 or later.
```

Author: Quick check before applying: v6.32.0 only covers LCP subparts (INP subparts need v6.33.0), so placing this note under the parent `### Diagnose Core Web Vitals with subparts` heading might suggest it's the minimum for both. Did you mean to place it under `#### Largest Contentful Paint subparts` instead? Happy to apply (and remove the existing note after the LCP table) once confirmed.

Reviewer: yes, i only meant to move it right under the heading!

Author: Applied, thanks!

> Largest Contentful Paint and Interaction to Next Paint break down into subparts, each isolating a specific phase of the metric. Use subpart data to identify which phase is responsible for a slow score.

Reviewer: ```suggestion
Largest Contentful Paint and Interaction to Next Paint break down into subparts, each isolating a specific phase of the metric. Use subpart data to identify which phase contributes most to a slow LCP or INP.
```

Author: Applied, thanks!

> Largest Contentful Paint and Interaction to Next Paint are broken down into subparts. Each subpart isolates a specific phase of the metric, helping you pinpoint where time is spent and where to focus optimization work.

Reviewer: ```suggestion
Largest Contentful Paint and Interaction to Next Paint break down into subparts, each isolating a specific phase of the metric. Use subpart data to identify which phase is responsible for a slow score.
```

Author: Applied, thanks!

#### Largest Contentful Paint subparts

> LCP is decomposed into four phases. Time to First Byte is reported separately as `view.first_byte`. The other three are nested under `view.performance.lcp.sub_parts`:

Reviewer: ```suggestion
LCP breaks down into four phases. Time to First Byte is collected separately as `view.first_byte`. The remaining three subparts are collected under `view.performance.lcp.sub_parts`:
```

Author: Applied, thanks!

> | Phase                  | Attribute                                              | What it represents                                                                                                                                                                |

Reviewer: ```suggestion
| Phase                  | Attribute                                              | Description                                                                                                                                                                |
```

Author: Applied, thanks!

|------------------------|--------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | Time to First Byte     | `view.first_byte`                                      | Time until the browser receives the first byte of the HTML response. Reflects server, CDN, and redirect time.                                                                     |

Reviewer: ```suggestion
| Time to First Byte     | `view.first_byte`                                      | Time until the browser receives the first byte of the HTML response, including server response time, CDN latency, and redirect time.                                                                     |
```

Author: Applied, thanks!

| Resource load delay    | `view.performance.lcp.sub_parts.load_delay`            | Time between TTFB and the start of the LCP resource load. Reflects priority hints, preloads, and render-blocking resources. `0` when the LCP element does not require a resource. |

> | Resource load time     | `view.performance.lcp.sub_parts.load_time`             | Time to load the LCP resource. Reflects image format, compression, and network conditions. `0` when the LCP element does not require a resource.                                  |

Reviewer: ```suggestion
| Resource load time     | `view.performance.lcp.sub_parts.load_time`             | Time to load the LCP resource, affected by image format, compression, and network conditions. `0` when the LCP element does not require a resource.                                  |
```

Author: Applied, thanks!

> | Render delay           | `view.performance.lcp.sub_parts.render_delay`          | Time between the LCP resource finishing loading and the LCP element being painted. Reflects long tasks and blocking JavaScript or CSS.                                            |

Reviewer: ```suggestion
| Render delay           | `view.performance.lcp.sub_parts.render_delay`          | Time between the LCP resource finishing loading and the LCP element being painted. High values typically indicate long tasks or blocking JavaScript or CSS.                                            |
```

Author: Applied, thanks!

> | Render delay           | `view.performance.lcp.sub_parts.render_delay`          | Time between the LCP resource finishing loading and the LCP element being painted. High values typically indicate long tasks or blocking JavaScript or CSS.                                            |

Reviewer: ```suggestion
| Render delay           | `view.performance.lcp.sub_parts.render_delay`          | Time between the LCP resource finishing loading and the LCP element being painted. High values indicate long tasks or blocking JavaScript or CSS.                                            |
```

Author: Applied, thanks!

> **Note**: Requires Browser SDK v6.32.0 or later.

Reviewer: ```suggestion
```

Author: Applied, thanks!

#### Interaction to Next Paint subparts

> 

Reviewer: ```suggestion
**Note**: These attributes require Browser SDK v6.33.0 or later.
```

Author: Applied, thanks!

> INP is decomposed into three phases nested under `view.performance.inp.sub_parts`:

Reviewer: ```suggestion
INP breaks down into three phases, collected under `view.performance.inp.sub_parts`:
```

Author: Applied, thanks!

> | Phase                  | Attribute                                              | What it represents                                                                                                                                                                |

Reviewer: ```suggestion
| Phase                  | Attribute                                              | Description                                                                                                                                                                |
```

Author: Applied, thanks!

|------------------------|--------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | Input delay            | `view.performance.inp.sub_parts.input_delay`           | Time from the user input until the event handler starts running. Reflects main-thread contention and long tasks.                                                                  |

Reviewer: ```suggestion
| Input delay            | `view.performance.inp.sub_parts.input_delay`           | Time from the user input until the event handler starts running. High values indicate main-thread contention and long tasks.                                                                  |
```

Author: Applied, thanks!

> | Processing duration    | `view.performance.inp.sub_parts.processing_duration`   | Duration of the event handler execution. Reflects handler complexity and synchronous work.                                                                                        |

Reviewer: ```suggestion
| Processing duration    | `view.performance.inp.sub_parts.processing_duration`   | Duration of the event handler execution. High values indicate complex or synchronous handler work.                                                                                        |
```

Author: Applied, thanks!

> | Presentation delay     | `view.performance.inp.sub_parts.presentation_delay`    | Time the browser took to render the next frame. Reflects style, layout, paint, and composite cost.                                                                                |

Reviewer: ```suggestion
| Presentation delay     | `view.performance.inp.sub_parts.presentation_delay`    | Time the browser took to render the next frame. High values indicate expensive style, layout, paint, or composite operations.                                                                                |
```

Author: Applied, thanks!

> **Note**: Requires Browser SDK v6.33.0 or later.

Reviewer: ```suggestion
```

Author: Applied, thanks!
